assembl/alembic/versions/7ea03fbce9a8_create_vote_session_table.py

```diff
@@ -0,0 +1,69 @@
+from alembic import context, op
+from sqlalchemy import Column, Integer, ForeignKey
+from sqlalchemy.schema import UniqueConstraint
+import transaction
+
+"""create vote_session table
+
+Revision ID: 7ea03fbce9a8
+Revises: c3f8bc9c75d5
+Create Date: 2017-12-22 13:28:30.634303
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '7ea03fbce9a8'
+down_revision = 'c3f8bc9c75d5'
+
+
+def LangStringId(column_name):
+    return Column(
+        column_name + '_id',
+        Integer(),
+        ForeignKey('langstring.id')
+    )
+
+
+def lang_strings_args(lang_strings_names):
```

Comment:
bad function name

Reply:
what would you recommend ? it's named like that because it's a function generating the langstrings arguments for the create_table function